demo/readStorage.js

```diff
@@ -0,0 +1,20 @@
+window.__storageAccess = {
```

Comment:
Merge this with the same file that Weiyi uses for reading inputs. I think it's read-inputs.js.